packages/protocol/contracts/thirdparty/onchainRA/utils/SigVerifyLib.sol

```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.0;
+
+import "../interfaces/ISigVerifyLib.sol";
+import "./RsaVerify.sol";
+import "./BytesUtils.sol";
+
+// Library for verifying signatures
+// Supports verifying signatures with the following algorithms:
+// - RS256
+// - ES256
+// - RS1
+
+contract SigVerifyLib is ISigVerifyLib {
```

Comment:
this is not a lib, I'd suggest to remove "Lib" from both `SigVerifyLib` and `ISigVerifyLib`.